src/test/java/com/exasol/adapter/dialects/elasticsearch/ElasticSearchSqlDialectTest.java

```diff
@@ -65,24 +65,41 @@ void testExasolSqlDialectSupportsAllCapabilities() {
                                 LAST_VALUE, STDDEV_POP, STDDEV_SAMP, VAR_POP, VAR_SAMP)),
                 () -> assertThat(capabilities.getScalarFunctionCapabilities(),
```

Comment:
I think this test duplicates the configuration.  It does not test any program logic. A `doesNotThrow` test would also give the coverage but avoid the duplication.

Reply:
I think the same. @redcatbear do you have any thought about it?

Reply:
It's true that this test only raises the coverage. But that's the point of it. The higher the coverage, the less likely you forgot to cover something more important.
There is no guarantee mind you, but it is still less likely.

I agree that covering each capability by using it would be better. That would also cover each enum entry. But that is not an easy feat. And it is done via integration tests mostly. All of that together in my eyes justify the duplication and little extra work.